scripts/fast-reboot-dump.py

```diff
@@ -216,9 +244,13 @@ def garp_send(arp_entries, map_mac_ip_per_vlan):
         sockets[src_if].bind((src_if, 0))
 
     # send arp packets
-    for vlan_name, dst_mac, dst_ip in arp_entries:
-        src_if = map_mac_ip_per_vlan[vlan_name][dst_mac]
-        send_arp(sockets[src_if], src_mac_addrs[src_if], src_ip_addrs[vlan_name], dst_mac, dst_ip)
+    for iface_name, dst_mac, dst_ip in arp_entries:
+        if iface_name in map_mac_ip_per_vlan:
+            src_if = map_mac_ip_per_vlan[iface_name][dst_mac]
+            send_vlan_arp(sockets[src_if], src_mac_addrs[src_if], src_ip_addrs[iface_name], dst_mac, dst_ip, iface_name)
```

Comment:
Anyway I don't understand why we need to send arp packets with dot1Q tags on non-trunk vlan interfaces?